### PR TITLE
Upgraded to scala 2.12.8 and twirl 1.4.1 so that it is usable with Java 11

### PR DESCRIPTION
**Pull request by dimovelev**

### CODE REVIEW

File: sample/templates/src/test/java/com/example/html/HelloTest.java [hunk 1]
```diff
@@ -7,6 +7,6 @@
 public class HelloTest {
   @Test public void helloWorld() {
     String actual = Hello.render("world").toString();
-    assertThat(actual).isEqualTo(String.format("%n<h1>Hello, world!</h1>%n"));
+    assertThat(actual).isEqualTo(String.format("<h1>Hello, world!</h1>\n"));
```
Comment (JakeWharton) [inline sample/templates/src/test/java/com/example/html/HelloTest.java]: Can you remove the `format` call?

File: sample/templates/src/test/java/com/example/html/IndexTest.java [hunk 2]
```diff
@@ -7,19 +7,18 @@
...
-        + "    <title>Index - Example</title>%n"
-        + "  </head>%n"
-        + "  <body>%n"
-        + "%n"
-        + "<h1>Hello, world!</h1>%n"
-        + "%n"
-        + "  </body>%n"
-        + "</html>%n"
-        + "%n");
+    String expected = String.format("<!DOCTYPE html>\n"
```
Comment (JakeWharton) [inline sample/templates/src/test/java/com/example/html/IndexTest.java]: Same here. We can drop the `format` call now.

File: sample/templates/src/test/java/com/example/html/NameListTest.java [hunk 3]
```diff
@@ -10,6 +10,6 @@
   @Test public void helloWorld() {
     List<String> names = Arrays.asList("One", "Two");
     String actual = NameList.render(names).toString();
-    assertThat(actual).isEqualTo(String.format("%n%n<h1>Hello, One!</h1>%n%n<h1>Hello, Two!</h1>%n%n"));
+    assertThat(actual).isEqualTo(String.format("\n<h1>Hello, One!</h1>\n\n<h1>Hello, Two!</h1>\n\n"));
```
Comment (JakeWharton) [inline sample/templates/src/test/java/com/example/html/NameListTest.java]: And here!

File: pom.xml [hunk 4]
```diff
@@ -44,36 +44,37 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java.version>1.8</java.version>
-    <twirl.version>1.3.4</twirl.version>
+    <java.version>1.10</java.version>
```
Comment (JakeWharton) [inline pom.xml]: Is this required for one of the dependency updates? I would prefer to keep source and target compatibility at 1.8 if possible.

**Comment by dimovelev** [comment]
Sorry - did not get any notifications from github. I removed the redundant formats and switched back the java compatibility to 1.8

**Comment by JakeWharton** [comment]
Squashed and merged as 497d7bdedbcd4c245629f046a3da32a76892886c. Will try to get out a release today.

**Comment by JakeWharton** [comment]
Released as 1.2.0. Should be available in Maven central in an hour or so.

**Pull request closed without merging**